version/v1.go

```diff
@@ -8,17 +8,11 @@ import (
 // Version1Routes registers all routes for the v1 version
 func Version1Routes(r *mux.Router) {
 
-	// this doesn't need versioning, yet
+	//server info in json format
 	base.RegisterHealthRoutes(r)
```

Comment:
Why do we have two `base.RegisterHealthRoutes(r)` one on `line 12` and another on `line 17`?

Reply:
You said the first endpoint should be "/" while the second should be "v1/health".  If I use just base.RegisterHealthRoutes(route) in line 17, the first enpoint would be now be "/v1". Should it be done that way?

Reply:
I'm wonder why they are both health routes. One is health and the other is home. So, why group them the same way?

Reply:
I had no idea I was supposed to group them separately, I will correct it. 
Thank you.

Reply:
In controller folder, what can I call the name of the handler for the first endpoint? I named the second handler as "HealthInfo"

Reply:
For now, I left it as **HealthCheck**

Reply:
> I'm wondering why they are both health routes. One is health and the other is home. So, why group them the same way?

here's my message above...I think you find a name from it.

Reply:
okay

Reply:
one is Health and one is Home

```go
func HealthCheck() {}

func Home() {}
```

Reply:
Done that. Thank youu.